static/js/compose.js

```diff
@@ -637,6 +643,40 @@ exports.initialize = function () {
         }
     });
 
+    // Show a warning if a private stream is linked
+    $(document).on('streamname_completed.zulip', function (event, data) {
+        if (compose_state.get_message_type() !== 'stream') {
+            return;
+        }
+
+        if (data !== undefined && data.stream !== undefined) {
+            var invite_only = data.stream.invite_only;
+            var name = data.stream.name;
+
+            if (invite_only === true) {
```

Comment:
Nitpick: Unnecessary `=== true`.

Reply:
I am getting eslint error if I make it `== true`

 `656:29  error  Expected '===' and instead saw '=='  eqeqeq`

Reply:
@rheaparekh You don't need `=== true` at all. `if (intvite_only)` should work just the same.

Reply:
Oh yes of course, my bad. I'll update it. Thank you for the review!